routes/session.js

```diff
@@ -118,7 +118,7 @@ function sessionRoutes (server, options, next) {
           })
         }
 
-        throw error
+        // throw error
```

Comment:
I likely did that while I was getting my bearings, left in error.  Sorry about that.